Makefile

```diff
@@ -377,7 +377,7 @@ deploy-test: _verify_cluster generate-keys update-local-image
 	$(MAKE) deploy-wiremock deploy-all
 
 # An alias for the deploy-test target
-deploy-service-for-subsystem-test: deploy-test enable-kube-api-for-subsystem
+deploy-service-for-subsystem-test: deploy-test
```

Comment:
Why?
I think this will break the operator-ztp-converged test

Reply:
Users who want to deploy the service for kube-api subsystem can follow [these](https://github.com/openshift/assisted-service/blob/a314b89e470e47ccc91651ae6775f3727bf95870/docs/dev/running-test.md?plain=1#L60-L65) instructions

You can't run it all the time because otherwise you break [the instructions](https://github.com/openshift/assisted-service/blob/a314b89e470e47ccc91651ae6775f3727bf95870/docs/dev/running-test.md?plain=1#L54-L58) for regular subsystem tests